scripts/extract_jartic_2024_2025.py

```
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+from pathlib import Path
+import time
+
+def process_jartic_archives():
```

Ruff (0.12.2)

`process_jartic_archives` is too complex (11 > 10)

(C901)